package/lib/src/controls/create_control.dart

```diff
@@ -220,6 +221,12 @@ Widget createControl(Control? parent, String id, bool parentDisabled) {
               control: controlView.control,
               children: controlView.children,
               parentDisabled: parentDisabled);
+        case "date_picker":
```

Comment:
Let's use the same naming convention for control names as the rest - lowecase and without `_` or `-` inside.